src/frontend/src/pages/ChangeRequestDetailPage/ReviewChangeRequestView.tsx

```diff
@@ -77,6 +88,18 @@ const ReviewChangeRequestsView: React.FC<ReviewChangeRequestViewProps> = ({
   const renderProposedSolutionModal: (scr: StandardChangeRequest) => JSX.Element = (scr: StandardChangeRequest) => {
     return (
       <Dialog fullWidth maxWidth={dialogWidth} open={modalShow} onClose={onHide} style={{ color: 'black' }}>
+        <IconButton
```

Comment:
the only thing that i have for this and i dont think it matters at all is placing this in the dialog title only because thats where I put it in the tab modal. But again this doesnt really matter since were placing it with an absolute